meta/runtime.yml

```diff
@@ -1,94 +1,8 @@
 ---
-requires_ansible: ">=2.15.0"
+requires_ansible: ">=2.16.0"
 plugin_routing:
   action:
     grpc_config:
       redirect: ansible.netcommon.grpc
     grpc_get:
       redirect: ansible.netcommon.grpc
-  filter:
```

Comment:
was this unintentional?

Reply:
These are set of deprecation, they were only serving are redirects, here no file removals